src/tests/ftest/util/agent_utils.py

```diff
@@ -88,13 +88,14 @@ def run_agent(basepath, server_list, client_list=None):
 
     for client in client_list:
         sessions[client] = subprocess.Popen(
-            ["ssh", client, "{} -i".format(daos_agent_bin)],
+            ["ssh", client, "-o ConnectTimeout=10",
+	    "{} -i".format(daos_agent_bin)],
```

Comment:
Doesn't this need to be indented by two more spaces?

Reply:
That is correct I hit return and it looked correct but CI seems to think it looks differently.

Reply:
Actually it seems to think that the issue was that there was a tab not the number of  spaces. I'll push up what it suggested and see if it is fixed.